scripts/asciidoc2html.py

```diff
@@ -35,7 +35,8 @@
 
 from scripts import utils
 
-DOC_DIR = pathlib.Path("qutebrowser/html/doc")
+REPO_ROOT = pathlib.Path(__file__).parent.parent
```

Comment:
```suggestion
REPO_ROOT = pathlib.Path(__file__).parent.parent.resolve()
```

similar to eed0373feb1a1750aa834b9c8d3524ddb1833780

Reply:
`.parent.parent` and `parents[1]` should give the same I guess, do you have a preference?